cronjobs/src/commands/_git_export_lfs.py

```diff
@@ -0,0 +1,265 @@
+    session.mount("https://", adapter)
+    return session
+
+
+def github_lfs_batch_request(
+    auth_header: str,
+    objects: Iterable[dict[str, Any]],
+    operation: str,  # "upload" or "download",
+    repo_owner: str,
+    repo_name: str,
+    timeout: float = HTTP_TIMEOUT_BATCH_SECONDS,
+) -> dict[str, Any]:
+    """
+    Generic Git LFS Batch call.
+    objects: [{"oid": "<sha256-hex>", "size": <int>}, ...]
+
+    https://github.com/git-lfs/git-lfs/blob/main/docs/api/batch.md#requests
+    """
+    url = f"https://github.com/{repo_owner}/{repo_name}.git/info/lfs/objects/batch"
+    headers = {
+        "Accept": "application/vnd.git-lfs+json",
+        "Content-Type": "application/vnd.git-lfs+json",
+        "Authorization": auth_header,
+    }
+    payload = {
+        "operation": operation,
+        "transfers": ["basic"],
+        "objects": list(objects),
+    }
+    session = _new_retrying_session()
+    r = session.post(url, headers=headers, json=payload, timeout=timeout)
+    if r.status_code != 200:
+        print(f"LFS: batch failed with status {r.status_code}: {r.text}")
+    r.raise_for_status()
+    return r.json()
+
+
+def _download_from_cdn_and_upload_to_lfs_volume(
+    source: tuple[str, int, str],  # (sha256_hex, size, src_url)
+    dest: tuple[str, str, dict[str, str]],  # (href, method, headers),
+    retry_max_count: int = HTTP_RETRY_MAX_COUNT,
+    retry_delay: float = HTTP_RETRY_DELAY_SECONDS,
+    timeout: float = HTTP_TIMEOUT_UPLOAD_SECONDS,
+) -> None:
+    """
+    Downloads the file locally (verifies digest/size), then uploads to specified URL.
+    """
+    sha256_hex, size, src_url = source
+    upload_href, upload_method, upload_headers = dest
+
+    with tempfile.NamedTemporaryFile() as tmp:
+        tmp_path = tmp.name
+        retry = 0
+        while retry < retry_max_count:
+            # Download from CDN into temp file.
+            downloaded_digest, downloaded_size = fetch_and_hash(
+                src_url, dest_file=tmp_path
+            )
+            if downloaded_size == size:
+                if downloaded_digest == sha256_hex:
+                    break
+            retry += 1
+            time.sleep(retry_delay)
+        else:
+            raise RuntimeError(
+                f"LFS: {src_url} failed to download after {retry_max_count} attempts"
+            )
+        # If we reach here, it means we successfully downloaded the file from the CDN.
+        # Now, upload it to LFS volume.
+        print(
+            f"LFS: uploading {src_url} -> {upload_method} {upload_href} ({size} bytes)"
+        )
+        with open(tmp_path, "rb") as f:
+            resp = requests.request(
+                upload_method.upper(),
+                upload_href,
+                data=f,
+                headers=upload_headers,
+                timeout=timeout,
+            )
+        if resp.status_code not in (200, 201, 204):
```

Comment:
```suggestion
        if not resp.ok:
```
Otherwise we may end up in a situation where we print the error message below without actually raising an exception, which would be confusing.